cli/internal/terraform/terraform/azure/main.tf

```diff
@@ -150,60 +152,19 @@ resource "azurerm_lb" "loadbalancer" {
 module "loadbalancer_backend_control_plane" {
   source = "./modules/load_balancer_backend"
 
-  name            = "${local.name}-control-plane"
-  loadbalancer_id = azurerm_lb.loadbalancer.id
-  ports = flatten([
-    {
-      name     = "bootstrapper",
-      port     = local.ports_bootstrapper,
-      protocol = "Tcp",
-      path     = null
-    },
-    {
-      name     = "kubernetes",
-      port     = local.ports_kubernetes,
-      protocol = "Https",
-      path     = "/readyz"
-    },
-    {
-      name     = "konnectivity",
-      port     = local.ports_konnectivity,
-      protocol = "Tcp",
-      path     = null
-    },
-    {
-      name     = "verify",
-      port     = local.ports_verify,
-      protocol = "Tcp",
-      path     = null
-    },
-    {
-      name     = "recovery",
-      port     = local.ports_recovery,
-      protocol = "Tcp",
-      path     = null
-    },
-    {
-      name     = "join",
-      port     = local.ports_join,
-      protocol = "Tcp",
-      path     = null
-    },
-    var.debug ? [{
-      name     = "debugd",
-      port     = local.ports_debugd,
-      protocol = "Tcp",
-      path     = null
-    }] : [],
-  ])
+  name                           = "${local.name}-control-plane"
+  loadbalancer_id                = azurerm_lb.loadbalancer.id
+  frontend_ip_configuration_name = azurerm_lb.loadbalancer.frontend_ip_configuration[0].name
+  ports                          = local.ports
```

Comment:
have you tested that its okay to pass a map with superfluous keys (e.g. priority)

Reply:
Yes, I've tested it.